tracer/src/Datadog.Trace/ClrProfiler/IntegrationType.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="IntegrationType.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal enum IntegrationType
```

Comment:
same comments about naming from the other `IntegrationType`

Reply:
changed in 3490e0c